package.json

```diff
@@ -81,7 +81,8 @@
     "ua-parser-js": ">0.7.30 <0.8.0",
     "secp256k1": "5.0.1",
     "socks": "2.7.3",
-    "web3-utils": "4.2.1"
+    "web3-utils": "4.2.1",
+    "@polkadot/api": "14.1.1"
```

Comment:
https://github.com/paritytech/txwrapper-core/blob/main/packages/txwrapper-core/package.json

has a dep on `"@polkadot/api": "^14.0.1",` 

14.1.1 is fine, but 14.2.1 starts breaking stuff. 

Simply upgrading our stack to 14.2.1 didn't fix things. Maybe it will but the coins team will need to investigate the changes that need to be made.

Reply:
its usually some combination between all the libs that belong to polkadot, they use ^ internally and do breaking changes with minor versions, causing any "compatible" lib upgrades not really compatible